tgui/packages/tgui/interfaces/FoodCart.tsx

```diff
@@ -0,0 +1,488 @@
+                bold>
+                {/* Get name */}
+                {capitalize(item.name)}
+              </Table.Cell>
+              <Table.Cell
+                textAlign="right">
+                {/* Get amount of item in storage */}
+                {item.quantity} in storage
+              </Table.Cell>
+              <Table.Cell>
+              {/* Make dispense button */}
+              <Button
+              fluid
+              content="Dispense"
+              textAlign="center"
+              fontSize="16px"
+              // Dissable if there is none of the item in storage
+              disabled={(
+                item.quantity === 0
+              )}
+              onClick={() => act("dispense", {
+                itemPath: item.type_path,
+              })}
+              />
+              </Table.Cell>
+          </Table.Row>
+      ))}
+      </Table>
+    );
+  } else {
+    return (
+      <Table>
+        <Table.Row>
+          <Table.Cell
+          fontSize="14px"
+          textAlign="center"
+          bold>
+            Food Storage Empty
+          </Table.Cell>
+        </Table.Row>
+      </Table>
+    );
+  }
+};
+
+const DrinkTab = (props, context) => {
+  // Get data from ui_data in backend code
+  const { data } = useBackend<Data>(context);
+  // Get needed variable for StorageRow
+  const { storage } = data;
+
+  // For organizing the Drink tab's information
+  return (
+    <Stack vertical>
+      <Stack.Item>
+        <Flex>
+          <Flex.Item
+          grow
+          mr={1}>
+            <Section
+            title="Glass Storage"
+            textAlign="center">
+              <StorageRow
+              quantity={storage.glass_quantity}
+              capacity={storage.glass_capacity} />
+            </Section>
+          </Flex.Item>
+          <Flex.Item
+          grow>
+            <Section
+            title="Drink Capacity"
+            textAlign="center">
+              <StorageRow
+              quantity={storage.drink_quantity}
+              capacity={storage.drink_capacity} />
+            </Section>
+          </Flex.Item>
+        </Flex>
+      </Stack.Item>
+      <Stack.Item>
+        <Section
+        title="Transfer Amount"
+        textAlign="center"
+        buttons={<Button
+          circular
+          tooltip="Amount of reagents to be transfered or purged"
+          icon="info" />}
+        >
+          <DrinkTransferRow />
+        </Section>
+      </Stack.Item>
+      <Stack.Item>
+        <Section
+        title="Cart Drink Storage"
+        textAlign="center"
+        buttons={<Button
+          circular
+          tooltip="Main reagent storage for the cart"
+          icon="info" />}>
+          <MainDrinkRow />
+        </Section>
+      </Stack.Item>
+      <Stack.Item>
+        <Section
+        title="Cart Mixer Storage"
+        textAlign="center"
+        buttons={<Button
+          circular
+          tooltip="Reagents to be poured into drinking glasses"
```

Comment:
Could you add to the Cart Mixer Storage tooltip that reagents will mix after pouring the glass? It may not be obvious where in other cases this is expected (Hoochmaster, chemmaster, e.g.)

Reply:
I did it for both storage tooltips